src/components/drawer/drawer.tsx

```diff
@@ -39,10 +39,19 @@ export const Drawer = ({
     }
 
     return (
-        <div onClick={handleClose} className={styles.background}>
+        <div
```

Comment:
this doesn't look like fixing lint issues

Reply:
it doesn't look like, but it is

<img width="850" alt="image" src="https://github.com/user-attachments/assets/1e623a03-dc32-4e7f-b434-1736f1af23be">

Reply:
use a button for a menu backdrop?

Reply:
sorry,  I thought  it was a button which opens the cart